week-3/reading-list/script.js

```diff
@@ -21,3 +22,23 @@ const books = [
   },
 ];
 
+
+const getReadingList = document.getElementById("content");
+const listOfBooks = document.getElementById("reading-list");
+console.log(getReadingList);
+
+for (const item of books){
+  const li = document.createElement("li");
+  listOfBooks.appendChild(li);
+
+  const p = document.createElement("p");
+  li.appendChild(p);
+  p.textContent = `${item.title} by ${item.author}`;
+
+  const image = document.createElement("img");
+  li.appendChild(image);
+  image.src = item.bookCoverImage;
+
+  if (item.alreadyRead === true){li.setAttribute("class", "green")}
+  else {li.setAttribute("class", "red")}
```

Comment:
Is more readable and good practice to declare all variables the to and then assign them in the middle and then append them at the end. So can you please rearrange all of your code in that order please?